### PR TITLE
Side nav

### PR DESCRIPTION
**Pull request by m-schwald**

### CODE REVIEW

File: client/src/components/Group.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+export default function Group({ openGroup, setOpenGroup }) {
+  return (
+    <NavContainer openGroup={openGroup}>
+      <IconGroup />
+      <h4> Groupname </h4>
+      <br />
+      <p> Berlin</p>
+      <p>
+        Apples ducks straw, quail a ostriches donkey, hay hook cucumbers. Lamb
+        pig rooster sheep. . Onion organic orange
```
Comment (miriamhaenle) [inline client/src/components/Group.js:18]: 😹

File: client/src/components/Group.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <Flexbox>
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+      </Flexbox>
+
+      <Button>
+        <Link to="./editGroup" onClick={() => setOpenGroup(!openGroup)}>
```
Comment (miriamhaenle) [inline client/src/components/Group.js:29]: Interesting, does that work with the `.` before the slash? 🤔

File: client/src/components/Group.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+`;
+
+const NavContainer = styled.div`
+  padding: 3rem 2rem;
+  position: absolute;
+  background: var(--onetransparent);
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100vw;
```
Comment (miriamhaenle) [inline client/src/components/Group.js:52]: I'm not sure but is it maybe obsolete. Doesn't `left: 0` and `right: 0`have the same effect as 100vw as it puts the content to 0 on both left and right? 🤔

File: client/src/components/Group.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  overflow: hidden;
+`;
+const Link = styled(NavLink)`
+  color: var(--dark);
+  text-align: center;
+  text-decoration: none;
+`;
+
+Group.propTypes = {
+  openGroup: bool.isRequired,
```
Comment (miriamhaenle) [inline client/src/components/Group.js:72]: What about the `setOpenGroup` prop? :)

File: client/src/components/Logo.js [hunk 5]
```diff
@@ -1,12 +1,22 @@
 import styled from "styled-components";
+import { bool, func } from "prop-types";
```
Comment (miriamhaenle) [inline client/src/components/Logo.js:2]: Nice, didn't know they can be import that way!

File: client/src/components/Nav.js [hunk 6]
```diff
@@ -1,22 +1,63 @@
...
-      <GroupBox>
-        <IconGroup />
-        <NavLink>groupname</NavLink>
-      </GroupBox>
-      <GroupBox>
-        <IconUser />
-        <NavLink>username</NavLink>
-      </GroupBox>
-    </Navi>
+    <>
```
Comment (miriamhaenle) [inline client/src/components/Nav.js:27]: No need for the additional React Fragments as the `<Navi>` wraps around the whole content

File: client/src/components/Profile.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  transition: ease-in-out 0.5s all;
+`;
+const Link = styled(NavLink)`
+  color: var(--dark);
+  text-align: center;
+  text-decoration: none;
+`;
+
+Profile.propTypes = {
+  openProfile: bool.isRequired,
```
Comment (miriamhaenle) [inline client/src/components/Profile.js:55]: What about `setOpenProfile`?

File: client/src/components/SideNav.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  padding: 0.2rem;
+  z-index: 10;
+  border-radius: 10px;
+  min-width: 80%;
+  text-align: center;
+  text-decoration: none;
+`;
+
+SideNav.propTypes = {
+  openNav: bool.isRequired,
```
Comment (miriamhaenle) [inline client/src/components/SideNav.js:57]: setOpenNav?

File: client/src/pages/Product.js [hunk 9]
```diff
@@ -13,7 +14,7 @@ export default function Product() {
     <>
```
Comment (miriamhaenle) [inline client/src/pages/Product.js:14]: No need for the React Fragment. `<Container> wraps it all

File: client/src/pages/offering.js [hunk 10]
```diff
@@ -1,18 +1,22 @@
...
 import Button from "../components/Button";
 import CardOffering from "../components/CardOffering";
 import ContainerFlat from "../components/ContainerFlat";
+import H3 from "../components/H3";
 
 import search from "../assets/images/search.svg";
 
 export default function Offering() {
   return (
     <>
```
Comment (miriamhaenle) [inline client/src/pages/offering.js:13]:
```suggestion
    
```

**Review by miriamhaenle — COMMENTED** [review]
🎉  Very nice job!

File: client/src/components/Group.js (hunk 2 above)
Comment (m-schwald) [reply client/src/components/Group.js:29]: it does. but I changed it and it still works.

File: client/src/components/Group.js (hunk 3 above)
Comment (m-schwald) [reply client/src/components/Group.js:52]: yeah, you're right

File: client/src/components/Group.js (hunk 4 above)
Comment (m-schwald) [reply client/src/components/Group.js:72]: check

File: client/src/components/Logo.js (hunk 5 above)
Comment (m-schwald) [reply client/src/components/Logo.js:2]: auto-import :)

**Pull request merged**